### ISSUE TITLE
Toolbar pagination not working

### ISSUE THREAD

**Issue by abettke** [react-pdf-viewer/react-pdf-viewer#1260]
Decided to test out the viability of this library and everything works real well. The only thing I can't seem to get working correctly is the pagination. The buttons do not change the page and clicking on the thumbnails also does not change the page. Below is the setup.

```
"@react-pdf-viewer/core": "^3.7.0"
"@react-pdf-viewer/default-layout": "^3.7.0"
"pdfjs-dist": "2.15.349"

```

```
import React from "react";
import format from "@app/lib/format";
import { Viewer, Worker } from "@react-pdf-viewer/core";
import { defaultLayoutPlugin } from "@react-pdf-viewer/default-layout";

import "@react-pdf-viewer/core/lib/styles/index.css";
import "@react-pdf-viewer/default-layout/lib/styles/index.css";

export const PdfViewer = ({ fileId }) => {
    const fileUrl = format.file(fileId);
    const layoutPlugin = defaultLayoutPlugin();

    return (
        <Worker workerUrl={"https://unpkg.com/pdfjs-dist@2.15.349/build/pdf.worker.min.js"}>
            <Viewer fileUrl={fileUrl} plugins={[layoutPlugin]} />
        </Worker>
    );
};
```

**Comment by phuocng** [comment]
@abettke Can you fork this [sample project](https://codesandbox.io/s/react-pdf-viewer-forked-yc8ut) and give me the link to see the issue?

**Comment by abettke** [comment]
Unable to reproduce off of that fork. Seems to work just fine, however integrating into our existing application and the only feature that doesn't work is the pagination. The events don't seem to fire `onClick` of those buttons. Any advice on tracking down what might be going on in this environment specifically?

**Comment by abettke** [comment]
Closing as I have figured out the issue. The problem was actually with the way that the viewer was being rendered by our page. Due to some auto-sizing from flex-boxes on the parent, the parent would allow the viewer to grow until it fit the space. The scrolling would then only happen on the outer parent container instead of within the viewer itself! 

Pagination now works again but nesting the `Viewer` component inside a div with fixed height. Closing this ticket now.

Great work on this library!
